index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Website</title>
+  </head>
+  <body>
+    <header>
+      <a class="name">Muh Noman</a>
+      <img class="navi" src="/images/navi.png" alt="navigation" />
```

Comment:
- [ ] please note that this `img` tag should be wrapped in a `nav` tag. please kindly make this change.

Reply:
@MrBrN197 Hi bro! I add this tag!